release.md

```diff
@@ -0,0 +1,12 @@
+**Design system - build and deploy HOWTO**
+
+To deploy package to the npm you need to be logged in. To confirm that you are logged in use you can use 'npm whoami'. If you are not use npm login and follow instructions.
+
+After that run:
+
+- 'npm run check' - this fires up linter and unit tests that are provided per package
+- 'npm run prettier' - to format code across the packages
```

Comment:
We could simply include these two in `predeploy` script.

Reply:
👌🏼